include/pqxx/connection.hxx

```diff
@@ -662,7 +663,7 @@ private:
    *
    * Returns the number of bytes written, including the trailing zero.
    */
-  size_t esc_to_buf(std::string_view text, char *buf) const;
+  std::size_t esc_to_buf(std::string_view text, char *buf) const;
```

Comment:
You might consider moving the changes from size_t to std::size_t to their own PR, to simplify review of the streaming code.

Reply:
Yes, normally I would have.  But this is not a regular PR.  It's basically an experimental new line of development, and it wasn't meant to stick around for this long.  Some unrelated stuff got caught up.

I do miss Bzr...  Git is just _awful_ for this stuff!